devenv/zabbix62/docker-compose.yml

```diff
@@ -3,7 +3,7 @@ version: "3"
 services:
   # Grafana
   grafana:
-    image: grafana/grafana:8.4.7
+    image: grafana/grafana:10.1.2
```

Comment:
there are multiple devenv-docker-compose files with `grafana` and `docker` sections.. i'd recommend that we either adjust all of the files, or we do not adjust any of them (and do the adjustments in a sepearate PR). WDYT?